src/main/java/io/wisetime/connector/jira/JiraDao.java

```diff
@@ -134,10 +136,19 @@ boolean hasConfiguredTimeZone() {
         );
   }
 
-  List<Issue> findIssuesOrderedById(final long startIdExclusive, final int maxResults) {
-    return query().select("SELECT jiraissue.id, project.pkey, jiraissue.issuenum, jiraissue.summary, jiraissue.timespent "
+  List<Issue> findIssuesOrderedById(final long startIdExclusive, final int maxResults, String... projectKeys) {
+    String query = "SELECT jiraissue.id, project.pkey, jiraissue.issuenum, jiraissue.summary, jiraissue.timespent "
         + "FROM project INNER JOIN jiraissue ON project.id = jiraissue.project "
-        + "WHERE jiraissue.id > ? ORDER BY ID ASC LIMIT ?;")
+        + "WHERE jiraissue.id > ? ";
+
+    if (ArrayUtils.isNotEmpty(projectKeys)) {
+      query += String.format("AND project.pkey in (%s) ", Arrays.stream(projectKeys)
```

Comment:
Is there a way to provide the list of keys as a parameter instead of using `String.format`?